addons/point_of_sale/models/pos_session.py

```diff
@@ -126,12 +126,20 @@ def _compute_cash_all(self):
     @api.constrains('user_id', 'state')
     def _check_unicity(self):
         # open if there is no session in 'opening_control', 'opened', 'closing_control' for one user
-        if self.search_count([('state', 'not in', ('closed', 'closing_control')), ('user_id', '=', self.user_id.id)]) > 1:
+        if self.search_count([
+                ('state', 'not in', ('closed', 'closing_control')),
+                ('user_id', '=', self.user_id.id),
+                ('name', 'not like', '%RESCUE FOR%'),
+            ]) > 1:
             raise ValidationError(_("You cannot create two active sessions with the same responsible!"))
 
     @api.constrains('config_id')
     def _check_pos_config(self):
-        if self.search_count([('state', '!=', 'closed'), ('config_id', '=', self.config_id.id)]) > 1:
+        if self.search_count([
+                ('state', '!=', 'closed'),
+                ('config_id', '=', self.config_id.id),
+                ('name', 'not like', '%RESCUE FOR%'),
```

Comment:
I think the `%` signs are redundant with the default behavior of the `not like` operator, even if they won't break anything ;-)
Same thing in `_check_unicity`